src/main/resources/lib/ssb/statistic.ts

```diff
@@ -68,17 +80,64 @@ export function setupHandlers(socket: Socket, socketEmitter: SocketEmitter): voi
   })
 }
 
+function datasetIdsFromStatistic(statistic: Content<Statistics>): Array<string> {
+  const mainTableId: Array<string> = statistic.data.mainTable ? [statistic.data.mainTable] : []
+  const statisticsKeyFigureId: Array<string> = statistic.data.statisticsKeyFigure ? [statistic.data.statisticsKeyFigure] : []
+  const attachmentTablesFiguresIds: Array<string> = statistic.data.attachmentTablesFigures ? forceArray(statistic.data.attachmentTablesFigures) : []
+  return [...mainTableId, ...statisticsKeyFigureId, ...attachmentTablesFiguresIds]
+}
+
+function sourceListFromStatistic(statistic: Content<Statistics>): Array<TbmlSources> {
+  const datasetIds: Array<string> = datasetIdsFromStatistic(statistic)
+
+  const datasets: Array<DatasetRepoNode<TbmlData>> = datasetIds.reduce((acc: Array<DatasetRepoNode<TbmlData>>, contentId: string) => {
+    const dataset: DatasetRepoNode<TbmlData> | null = getDatasetFromContentId(contentId)
+    if (dataset) acc.push(dataset)
+    return acc
+  }, [])
+
+  const byOwners: any = groupBy((source: Source) => {
+    return `${source.owner}`
+  } )
+
+  return datasets.map((dataset) => {
+    return {
+      tbmlId: dataset._name,
+      sourceList: dataset.data && typeof(dataset.data) !== 'string' && dataset.data.tbml.metadata.sourceList ?
+        byOwners(dataset.data.tbml.metadata.sourceList) : undefined
+    }
+  })
+}
+
+function getDatasetFromContentId(contentId: string): DatasetRepoNode<TbmlData> | null {
+  const queryResult: QueryResponse<Highchart | Table | KeyFigure> = query({
+    query: `_id = '${contentId}'`,
+    count: 1,
+    filters: {
+      exists: {
+        field: 'data.dataSource.tbprocessor.urlOrId'
+      }
+    }
+  })
+
+  const content: Content<DataSource> | undefined = queryResult.count === 1 ? queryResult.hits[0] : undefined
+  return content ? getTbprocessor(content, 'master') : null
+}
+
 function prepStatistics(statistics: Array<Content<Statistics>>): Array<StatisticDashboard> {
   const statisticData: Array<StatisticDashboard> = []
   statistics.map((statistic: Content<Statistics>) => {
     const statregData: StatregData | undefined = statistic.data.statistic ? getStatregInfo(statistic.data.statistic) : undefined
-    if (statregData && statregData.nextRelease && moment(statregData.nextRelease).isSameOrAfter(new Date(), 'day')) {
+    const relatedTables: Array<TbmlSources> = sourceListFromStatistic(statistic)
+    //if (statregData && statregData.nextRelease && moment(statregData.nextRelease).isSameOrAfter(new Date(), 'day')) {
```

Comment:
Why is the check isSameOrAfter removed?

Reply:
I needed to get some content in the list. Should be changed back to normal the normal test.